se_feedback/sprint-5/feedback_sprint_5_all.md

```diff
@@ -0,0 +1,97 @@
+|      |            |
+|------|------------|
+|Group | Rubber Duck Debuggers |
+|Assignment|Sprint 5|
+|Date|29/05/16|
+|TA|Bastiaan Reijm|
+
+#Sprint Feedback
+Feedback and Grades for Sprint 5.
+
+Total: **9.55**
+
+| User Story | Score |
+|------------|-------|
+| definition |  9   |
+| splitting  |  10     |
+| responsibility | 10  |
+
+| Learning from History | Score |
+|-----------------------|-------|
+| estimation            | 10    |
+| prioritisation        | 10    |
+| reflection            | 9   |
+
+## Notes
+* Well done
+* That Matthias took over Robin's task should be added in a note, it looks like both were assigned responsibility and then one member was removed
+* Why is the row Octopeer from Logging Data empty and what is it?
+* Blank page 3
+* Make sure everyone catches up again on their hours in the next sprints.
```

Comment:
We are not sure what you mean by this. From our point of view, there are two options:

 - You have read the table, assuming this is a complete overview of the hours made. This is not the case, there is a footnote indicating this is only the time spent on the issues listed in the retrospective
 - This comment was on the external events. We have listed this item in the retrospective for completeness, but we did manage to make all the required hours (as was discussed in the meeting)